dev-libs/serdisplib/files/disable-static-build.patch

```diff
@@ -0,0 +1,86 @@
+--- a/server/Makefile.in	2018-05-15 21:52:38.966823295 +0200
++++ b/server/Makefile.in	2018-05-15 21:58:27.137821967 +0200
+@@ -83,13 +83,13 @@
+ .c.o:
+ 	$(CC) $(CFLAGS) $(DEFINES)  -c $(INCLUDES) $<
+ 
+-serdispd: $(LIB_DIR)/$(LIB_STATIC) $(SDISPDOBJECTS)
++serdispd: $(LIB_DIR)/$(LIB_SHARED) $(SDISPDOBJECTS)
+ 	$(CC) -o $@ $(SDISPDOBJECTS) $(LIB_SERDISP) $(EXTRA_LIBS_STATIC)
+ 
+-serdisplearn: $(LIB_DIR)/$(LIB_STATIC) $(SLEARNOBJECTS)
++serdisplearn: $(LIB_DIR)/$(LIB_SHARED) $(SLEARNOBJECTS)
+ 	$(CC) -o $@ $(SLEARNOBJECTS) $(LIB_SERDISP) $(EXTRA_LIBS_STATIC)
+ 
+-serdispproxy: $(LIB_DIR)/$(LIB_STATIC) $(SPROXYOBJECTS)
++serdispproxy: $(LIB_DIR)/$(LIB_SHARED) $(SPROXYOBJECTS)
+ 	$(CC) -o $@ $(SPROXYOBJECTS) $(LIB_SERDISP) $(EXTRA_LIBS_STATIC)
+ 
+ 
+--- a/src/Makefile.in	2018-05-15 21:52:56.268823229 +0200
++++ b/src/Makefile.in	2018-05-15 21:59:05.296821821 +0200
+@@ -167,7 +167,7 @@
+ 
+ all: libraries programs
+ 
+-libraries: library.static library.shared
++libraries: library.shared
+ 
+ library.static: $(LIB_DIR)/$(LIB_STATIC)
+ 
+@@ -188,8 +188,8 @@
+ 	cd $(LIB_DIR) && $(LN_S) -f $(LIB_SHARED) lib$(LIB_NAME).so.$(VERSION_MAJOR)
+ 	cd $(LIB_DIR) && $(LN_S) -f lib$(LIB_NAME).so.$(VERSION_MAJOR) lib$(LIB_NAME).so
+ 
+-testserdisp: $(LIB_DIR)/$(LIB_STATIC) $(OBJECTS)
+-	$(CC) -o testserdisp $(OBJECTS) $(LIB_SERDISP) $(EXTRA_LIBS_STATIC)
++testserdisp: $(LIB_DIR)/$(LIB_SHARED) $(OBJECTS)
++	$(CC) -o testserdisp $(OBJECTS) $(LIB_SERDISP) $(EXTRA_LIBS_SHARED)
```

Comment:
Did you miss this one or is this intentional?

Reply:
Grr.. no :( Thanks, fixed.